src/json.cpp

```diff
@@ -357,8 +357,8 @@ static void to_surrogate_pair(uint32_t unicode, uint16_t *uc, uint16_t *lc)
   *lc = (n & 0x3FF) | 0xDC00;
 }
 
-bool is_space        (const char *c);
-bool is_digit        (const char *c);
+bool is_space        (const char& c);
+bool is_digit        (const char& c);
```

Comment:
Yep, updated. I'm working through some inconsistencies in this as an experiment.